src/services/routing.ts

```diff
@@ -134,7 +134,7 @@ export function getPrimaryLabelFromUrl(): RouteProps {
   const routeParams = endOfUrl.split('/');
 
   let labelName = routeParams[0];
-  const labelValue = routeParams[1];
+  const labelValue = decodeURIComponent(routeParams[1]);
```

Comment:
There was a running condition where, for a moment, when you're building the "exploration prefix" based on route params, you could get a raw an encoded string:

<img width="824" height="266" alt="imagen" src="https://github.com/user-attachments/assets/4c6c131b-e6e2-46ce-9c9c-7766b8a95c03" />

Very niche running condition, surfaced by local Loki and `{service_name="\"grafana/data\""}`